.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "plugins/paulvonzimmerman/patreon"]
```

Comment:
Why is this added if we're not using a submodule in the end?

Reply:
I think I accidentally added it, I'll remove it.